app/controllers/claims/support/support_users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Claims::Support::SupportUsersController < Claims::Support::ApplicationController
+  def index
+    @users = Claims::User.support_users
+  end
+
+  def new
```

Comment:
If first name, last name, email params are available, you should initialise the form with them. See Jamie's work on the add user journey. Otherwise, the 'check page' just takes you back to an empty form which is a very unsatisfying journey.

Reply:
I've updated the flows to support adding default values via query params.